internal/c/libqb/src/glut-main-thread.cpp

```diff
@@ -178,8 +181,11 @@ void libqb_start_main_thread(int argc, char **argv) {
 // from two threads at the same time).
 //
 // This is accomplished by simply queuing a GLUT message that calls exit() for us.
-void libqb_exit(int exitcode)
-{
+void libqb_exit(int exitcode) {
+#ifdef QB64_MACOSX
+    macMouseDone();
```

Comment:
I think this call belongs in [here](https://github.com/QB64-Phoenix-Edition/QB64pe/blob/c7e912ace636e90248710196e4b189af8dfc8967/internal/c/libqb/src/glut-message.cpp#L52) instead. I suspect that you're intended to only call some of those `CG*` functions from the main thread.

Reply:
Moved this to `glut_message_exit_program::execute()`.